pkg/identity/cache.go

```diff
@@ -87,6 +92,10 @@ func identityWatcher(owner IdentityAllocatorOwner, events allocator.AllocatorEve
 		case kvstore.EventTypeModify:
 			// Ignore modify events
 		}
+		metrics.KVStoreOperationsTotal.With(map[string]string{
```

Comment:
Should this only be for Create and Delete events since we ignore modify events?

Reply:
No, the metrics will be reported that has been used, other thing is that the watcher handler does nothing.